docs/userguide/storagedriver/selectadriver.md

```diff
@@ -81,6 +81,12 @@ backing file system:
     |zfs            |Yes                           |
 
 
+**Note**
+Some storage drivers can not run over certain backing filesystem, details as below:
+
+1. overlay can not run over btrfs, aufs, zfs
+2. aufs can not run over btrfs, aufs
```

Comment:
Perhaps we should add an extra column to the table above (e.g. "incompatible with"; better names are welcome)

While you''re making those changes, could you fix the indentation of the table, because right now it's seen as a code-block, not a table;
<img width="633" alt="screen shot 2016-02-02 at 15 48 36" src="https://cloud.githubusercontent.com/assets/1804568/12752801/7a225476-c9c4-11e5-8964-e9f121db1720.png">